vm/actor/src/code.rs

```diff
@@ -2,10 +2,11 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use cid::Cid;
+use serde::{Deserialize, Serialize};
 
 /// CodeID is the reference to the code which is attached to the Actor state.
 /// There are builtin IDs and the option for custom code with a Cid
-#[derive(PartialEq, Eq, Clone, Debug)]
+#[derive(PartialEq, Eq, Clone, Debug, Serialize, Deserialize)]
```

Comment:
```suggestion
#[derive(PartialEq, Eq, Clone, Debug)]
```

Sorry one last quick change and good with me, just want to not be confusing that the CodeID will be serialized and that the format is checked